client/src/components/Jumbotron/index.js

```diff
@@ -3,7 +3,12 @@ import React from "react";
 function Jumbotron({ children }) {
   return (
     <div
-      style={{ height: 300, clear: "both", paddingTop: 120, textAlign: "center" }}
+      style={{
```

Comment:
extract this style object out and name it something so you have a description of what this style is for exactly. I dont know what this style is really for other than the parent div. and if you decide you want to use it later you can. Putting it super inline like this doesn't make it reusable at all.